src/theory/logic_info.cpp

```diff
@@ -118,12 +118,21 @@ bool LogicInfo::isHigherOrder() const
 }
 
 /** Is this the all-inclusive logic? */
-bool LogicInfo::hasEverything() const {
-  PrettyCheckArgument(d_locked, *this,
+bool LogicInfo::hasEverything(bool andHol) const
```

Comment:
I don't think we should have an argument here.  The use of hasEverything should always be agnostic to higher-order.

Reply:
I don't see how this assymetry impacts anything.  It is much clearer to me to have `hasEverything` not take an argument.

Reply:
It impacts what conceptually is going on. "Everything" means "ALL".